docker/envs/.env.docker.devnet

```
@@ -1,6 +1,18 @@
 # Default configuration for devnet docker-compose stack.
 # These values work out of the box — override by creating .env.docker.local.
 
+# ENSIndexer
 PLUGINS=subgraph,ensv2
+# ENSIndexer and ENSApi
+ENSINDEXER_SCHEMA_NAME=docker_devnet_v1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ENSINDEXER_SCHEMA_NAME key should go before the PLUGINS key

(UnorderedKey)